src/Utils/request/uploadFile.ts

```
@@ -43,13 +50,61 @@
     };
 
-    if (setUploadPercent != null) {
+    if (setUploadPercent != null || (options && options.setOverallProgress)) {
       xhr.upload.onprogress = (event: ProgressEvent) => {
-        handleUploadPercentage(event, setUploadPercent);
+        if (setUploadPercent) {
+          handleUploadPercentage(event, setUploadPercent);
+        }
+        if (
+          options &&
+          options.setOverallProgress &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)